Pluging Disabled Paper-592

hi unfortunately Titlemanager doesn't work if I run the server on the latest Paaper-Spigot version
up to V.5.8.4 it works fine
Here are the logs:
[08:44:08] [Server thread/ERROR]: Error occurred while enabling Craftconomy3 v3.5.3-SNAPSHOT (Is it up to date?)
java.lang.RuntimeException: java.lang.ClassNotFoundException: com.mysql.jdbc.jdbc2.optional.MysqlDataSource
	at com.zaxxer.hikari.util.UtilityElf.createInstance(UtilityElf.java:107) ~[?:?]
	at com.zaxxer.hikari.pool.PoolBase.initializeDataSource(PoolBase.java:327) ~[?:?]
	at com.zaxxer.hikari.pool.PoolBase.<init>(PoolBase.java:114) ~[?:?]
	at com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:108) ~[?:?]
	at com.zaxxer.hikari.HikariDataSource.<init>(HikariDataSource.java:81) ~[?:?]
	at com.greatmancode.craftconomy3.storage.sql.MySQLEngine.<init>(MySQLEngine.java:49) ~[?:?]
	at com.greatmancode.craftconomy3.storage.StorageHandler.<init>(StorageHandler.java:41) ~[?:?]
	at com.greatmancode.craftconomy3.Common.initialiseDatabase(Common.java:361) ~[?:?]
	at com.greatmancode.craftconomy3.Common.onEnable(Common.java:133) ~[?:?]
	at com.greatmancode.craftconomy3.tools.interfaces.BukkitLoader.onEnable(BukkitLoader.java:53) ~[?:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263) ~[patched_1.16.5.jar:git-Paper-592]
	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:380) ~[patched_1.16.5.jar:git-Paper-592]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:483) ~[patched_1.16.5.jar:git-Paper-592]
	at org.bukkit.craftbukkit.v1_16_R3.CraftServer.enablePlugin(CraftServer.java:501) ~[patched_1.16.5.jar:git-Paper-592]
	at org.bukkit.craftbukkit.v1_16_R3.CraftServer.enablePlugins(CraftServer.java:415) ~[patched_1.16.5.jar:git-Paper-592]
	at net.minecraft.server.v1_16_R3.MinecraftServer.loadWorld(MinecraftServer.java:580) ~[patched_1.16.5.jar:git-Paper-592]
	at net.minecraft.server.v1_16_R3.DedicatedServer.init(DedicatedServer.java:280) ~[patched_1.16.5.jar:git-Paper-592]
	at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:1055) ~[patched_1.16.5.jar:git-Paper-592]
	at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$a$0(MinecraftServer.java:289) ~[patched_1.16.5.jar:git-Paper-592]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_131]
Caused by: java.lang.ClassNotFoundException: com.mysql.jdbc.jdbc2.optional.MysqlDataSource
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381) ~[?:1.8.0_131]
	at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:182) ~[patched_1.16.5.jar:git-Paper-592]
	at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:111) ~[patched_1.16.5.jar:git-Paper-592]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424) ~[?:1.8.0_131]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357) ~[?:1.8.0_131]
	at com.zaxxer.hikari.util.UtilityElf.createInstance(UtilityElf.java:94) ~[?:?]


[08:45:32] [Server thread/ERROR]: Could not pass event PlayerJoinEvent to TitleManager v2.3.3
java.lang.NoClassDefFoundError: com/greatmancode/craftconomy3/Common
	at net.milkbowl.vault.economy.plugins.Economy_Craftconomy3.getBalance(Economy_Craftconomy3.java:279) ~[?:?]
	at net.milkbowl.vault.economy.plugins.Economy_Craftconomy3.getBalance(Economy_Craftconomy3.java:121) ~[?:?]
	at net.milkbowl.vault.economy.AbstractEconomy.getBalance(AbstractEconomy.java:22) ~[?:?]
	at io.puharesource.mc.titlemanager.internal.services.placeholder.PlaceholderServiceText$loadBuiltinPlaceholders$21.invoke(PlaceholderServiceText.kt:81) ~[?:?]
	at io.puharesource.mc.titlemanager.internal.services.placeholder.PlaceholderServiceText$loadBuiltinPlaceholders$21.invoke(PlaceholderServiceText.kt:29) ~[?:?]
	at io.puharesource.mc.titlemanager.internal.services.placeholder.PlaceholderServiceText$createPlaceholder$2.getText(PlaceholderServiceText.kt:192) ~[?:?]
	at io.puharesource.mc.titlemanager.internal.services.placeholder.PlaceholderServiceText.replaceText(PlaceholderServiceText.kt:152) ~[?:?]
	at io.puharesource.mc.titlemanager.internal.services.features.PlayerListServiceSpigot.setFooter(PlayerListServiceSpigot.kt:156) ~[?:?]
	at io.puharesource.mc.titlemanager.internal.services.features.PlayerListServiceSpigot$createFooterSendableAnimation$4.invoke(PlayerListServiceSpigot.kt:101) ~[?:?]
	at io.puharesource.mc.titlemanager.internal.services.features.PlayerListServiceSpigot$createFooterSendableAnimation$4.invoke(PlayerListServiceSpigot.kt:22) ~[?:?]
	at io.puharesource.mc.titlemanager.internal.model.animation.PartBasedSendableAnimation.update(PartBasedSendableAnimation.kt:90) ~[?:?]
	at io.puharesource.mc.titlemanager.internal.model.animation.PartBasedSendableAnimation.start(PartBasedSendableAnimation.kt:60) ~[?:?]
	at io.puharesource.mc.titlemanager.internal.services.features.PlayerListServiceSpigot.setProcessedFooter(PlayerListServiceSpigot.kt:171) ~[?:?]
	at io.puharesource.mc.titlemanager.internal.services.features.PlayerListServiceSpigot.setProcessedHeaderAndFooter(PlayerListServiceSpigot.kt:198) ~[?:?]
	at io.puharesource.mc.titlemanager.internal.services.event.ListenerServiceSpigot$registerSetHeaderAndFooter$$inlined$listenEventSync$1.invoke(ListenerServiceSpigot.kt:212) ~[?:?]
	at io.puharesource.mc.titlemanager.internal.services.event.ListenerServiceSpigot$registerSetHeaderAndFooter$$inlined$listenEventSync$1.invoke(ListenerServiceSpigot.kt:28) ~[?:?]
	at io.puharesource.mc.titlemanager.internal.model.event.TMEventListener$executor$1.execute(TMEventListener.kt:31) ~[?:?]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[patched_1.16.5.jar:git-Paper-592]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[patched_1.16.5.jar:git-Paper-592]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:607) ~[patched_1.16.5.jar:git-Paper-592]
	at net.minecraft.server.v1_16_R3.PlayerList.postChunkLoadJoin(PlayerList.java:356) ~[patched_1.16.5.jar:git-Paper-592]
	at net.minecraft.server.v1_16_R3.PlayerList.lambda$null$1(PlayerList.java:303) ~[patched_1.16.5.jar:git-Paper-592]
	at net.minecraft.server.v1_16_R3.PlayerConnection.tick(PlayerConnection.java:314) ~[patched_1.16.5.jar:git-Paper-592]
	at net.minecraft.server.v1_16_R3.NetworkManager.a(NetworkManager.java:408) ~[patched_1.16.5.jar:git-Paper-592]
	at net.minecraft.server.v1_16_R3.ServerConnection.c(ServerConnection.java:158) ~[patched_1.16.5.jar:git-Paper-592]
	at net.minecraft.server.v1_16_R3.MinecraftServer.b(MinecraftServer.java:1501) ~[patched_1.16.5.jar:git-Paper-592]
	at net.minecraft.server.v1_16_R3.DedicatedServer.b(DedicatedServer.java:418) ~[patched_1.16.5.jar:git-Paper-592]
	at net.minecraft.server.v1_16_R3.MinecraftServer.a(MinecraftServer.java:1328) ~[patched_1.16.5.jar:git-Paper-592]
	at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:1116) ~[patched_1.16.5.jar:git-Paper-592]
	at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$a$0(MinecraftServer.java:289) ~[patched_1.16.5.jar:git-Paper-592]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_131]
Caused by: java.lang.ClassNotFoundException: com.greatmancode.craftconomy3.Common
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381) ~[?:1.8.0_131]
	at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:182) ~[patched_1.16.5.jar:git-Paper-592]
	at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:111) ~[patched_1.16.5.jar:git-Paper-592]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424) ~[?:1.8.0_131]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357) ~[?:1.8.0_131]
Can you Help me ? All Plugins Up-To-Date

--- a/src/main/java/com/greatmancode/craftconomy3/converter/converters/Iconomy6.java
+++ b/src/main/java/com/greatmancode/craftconomy3/converter/converters/Iconomy6.java
@@ -120,10 +120,7 @@ public class Iconomy6 extends Converter {
         try {
             HikariConfig config = new HikariConfig();
             config.setMaximumPoolSize(10);
-            config.setDataSourceClassName("com.mysql.jdbc.jdbc2.optional.MysqlDataSource");
-            config.addDataSourceProperty("serverName", getDbConnectInfo().get("address"));
-            config.addDataSourceProperty("port", getDbConnectInfo().get("port"));
-            config.addDataSourceProperty("databaseName", getDbConnectInfo().get("database"));
+            config.setJdbcUrl("jdbc:mysql://" + getDbConnectInfo().get("address") + ":" + getDbConnectInfo().get("port") + "/" + getDbConnectInfo().get("database"));
             config.addDataSourceProperty("user", getDbConnectInfo().get("username"));
             config.addDataSourceProperty("password", getDbConnectInfo().get("password"));
             config.addDataSourceProperty("autoDeserialize", true);

--- a/src/main/java/com/greatmancode/craftconomy3/converter/converters/Mineconomy.java
+++ b/src/main/java/com/greatmancode/craftconomy3/converter/converters/Mineconomy.java
@@ -63,10 +63,7 @@ public class Mineconomy extends Converter {
         } else if ("mysql".equals(getSelectedDbType())) {
             HikariConfig config = new HikariConfig();
             config.setMaximumPoolSize(10);
-            config.setDataSourceClassName("com.mysql.jdbc.jdbc2.optional.MysqlDataSource");
-            config.addDataSourceProperty("serverName", getDbConnectInfo().get("address"));
-            config.addDataSourceProperty("port", getDbConnectInfo().get("port"));
-            config.addDataSourceProperty("databaseName", getDbConnectInfo().get("database"));
+            config.setJdbcUrl("jdbc:mysql://" + getDbConnectInfo().get("address") + ":" + getDbConnectInfo().get("port") + "/" + getDbConnectInfo().get("database"));
             config.addDataSourceProperty("user", getDbConnectInfo().get("username"));
             config.addDataSourceProperty("password", getDbConnectInfo().get("password"));
             config.addDataSourceProperty("autoDeserialize", true);

--- a/src/main/java/com/greatmancode/craftconomy3/storage/sql/MySQLEngine.java
+++ b/src/main/java/com/greatmancode/craftconomy3/storage/sql/MySQLEngine.java
@@ -35,10 +35,7 @@ public class MySQLEngine extends SQLStorageEngine {
     public MySQLEngine() {
         HikariConfig config = new HikariConfig();
         config.setMaximumPoolSize(Common.getInstance().getMainConfig().getInt("System.Database.Poolsize", 20));
-        config.setDataSourceClassName("com.mysql.jdbc.jdbc2.optional.MysqlDataSource");
-        config.addDataSourceProperty("serverName", Common.getInstance().getMainConfig().getString("System.Database.Address", "localhost"));
-        config.addDataSourceProperty("port", Common.getInstance().getMainConfig().getString("System.Database.Port", "3306"));
-        config.addDataSourceProperty("databaseName", Common.getInstance().getMainConfig().getString("System.Database.Db", "Craftconomy"));
+        config.setJdbcUrl("jdbc:mysql://" + Common.getInstance().getMainConfig().getString("System.Database.Address", "localhost") + ":" + Common.getInstance().getMainConfig().getString("System.Database.Port", "3306") + "/" + Common.getInstance().getMainConfig().getString("System.Database.Db", "Craftconomy"));
         config.addDataSourceProperty("user", Common.getInstance().getMainConfig().getString("System.Database.Username", "root"));
         config.addDataSourceProperty("password", Common.getInstance().getMainConfig().getString("System.Database.Password", ""));
         config.addDataSourceProperty("autoDeserialize", true);

--- a/src/main/java/com/greatmancode/craftconomy3/utils/OldFormatConverter.java
+++ b/src/main/java/com/greatmancode/craftconomy3/utils/OldFormatConverter.java
@@ -55,10 +55,7 @@ public class OldFormatConverter {
         HikariConfig config = new HikariConfig();
         if (dbType.equalsIgnoreCase("mysql")) {
             config.setMaximumPoolSize(10);
-            config.setDataSourceClassName("com.mysql.jdbc.jdbc2.optional.MysqlDataSource");
-            config.addDataSourceProperty("serverName", Common.getInstance().getMainConfig().getString("System.Database.Address", "localhost"));
-            config.addDataSourceProperty("port", Common.getInstance().getMainConfig().getString("System.Database.Port", "3306"));
-            config.addDataSourceProperty("databaseName", Common.getInstance().getMainConfig().getString("System.Database.Db", "Craftconomy"));
+            config.setJdbcUrl("jdbc:mysql://" + Common.getInstance().getMainConfig().getString("System.Database.Address", "localhost") + ":" + Common.getInstance().getMainConfig().getString("System.Database.Port", "3306") + "/" + Common.getInstance().getMainConfig().getString("System.Database.Db", "Craftconomy"));
             config.addDataSourceProperty("user", Common.getInstance().getMainConfig().getString("System.Database.Username", "root"));
             config.addDataSourceProperty("password", Common.getInstance().getMainConfig().getString("System.Database.Password", ""));
             config.addDataSourceProperty("autoDeserialize", true);

